spec/tic_tac_toe_spec.rb

```diff
@@ -1,13 +1,16 @@
 require 'spec_helper'
 require './lib/tic_tac_toe'
+require './lib/board.rb'
 
 describe TicTacToe do
   before(:each) do
     @test_game = TicTacToe.new
   end
 
   it 'can access draw_board from Board class' do
```

Comment:
I see you have methods `draw_board` and `display_board` - you list one in the test name and a different one in the expect. Which one are you testing here?

Reply:
Ah I'm testing display because it calls "draw_board", but I suspect that when I refactor board it will be moved out of the board class all together...

Reply:
I'll change the language though! It's a good call out